WTWSMS/history/characters/greek.txt

```diff
@@ -3625,4 +3625,31 @@
 	434.1.1={
 		death=yes
 	}
-}
\ No newline at end of file
+}
+1310010= {
+	name="Alexander"
+	dynasty=1051146
+	religion="nicene"
+	culture="greek"
+	trait="mastermind_theologian"
+	trait="zealous"
+	trait="kind"
+	472.1.1={
+		birth=yes
+	}
+	535.1.1={
+		death=yes
+	}
+}
+1310011= {
+	name="Mauricius"
```

Comment:
Similarly, wouldn't this be the Greek version?

Reply:
Same as above

Reply:
Everything solved again, I think